README.md

```diff
@@ -4,6 +4,15 @@ A project based learning activity for people who are getting started with Git an
 
```

Comment:
What is the lowest and highest response times for the different input devices 3mouse, touch, key, etc.?